pyton_hw.py

```diff
@@ -0,0 +1,47 @@
+import random
+
+
+def play_coin_flip():
+    player_wins = 0
+    player_losses = 0
+
+    while True:
+        player_choice = input("Виберіть Орел (O) або Решка (Р), або натисніть (X) щоб завершити гру: ")
+        player_choice = player_choice.upper()
+
+        if player_choice == 'X':
+            break
+
+        while player_choice not in ['O', 'P']:
+            player_choice = input("Виберіть Орел (O) або Решка (Р), або натисніть (X) щоб завершити гру: ")
```

Comment:
line 9-10 and line 16-17 - is a duplications of code. Move it to a separate function and re-use